template-parts/single-govpack_issues.php

```diff
@@ -0,0 +1,61 @@
+<?php // phpcs:ignore WordPress.Files.FileName.NotHyphenatedLowercase
```

Comment:
This is almost identical to `single-govpack_profiles.php` and should be refactored to reduce duplication.

Reply:
This has been refactored into `single-govpack.php`, which checks for the current post type and pulls the correct stories.